### DISCUSSION TITLE
Disable Format on save

### DISCUSSION THREAD

**Discussion by radstevee** [LazyVim/LazyVim#4457] [Q&A] [answered]
It often makes my code less readable with C stuff, e.g. dwm and inconsistent formatting, like this:
![image](https://github.com/user-attachments/assets/2ae9b050-761a-439e-9b0b-ea9698b1a0bc)

How can I disable it?

**Comment by dpetka2001** [comment]
`<leader>uf` is the keymap to toggle it off for your current Neovim session.

**Reply by radstevee** [reply]
Is there a way to disable it globally, for all sessions?

**Reply by dpetka2001** [reply, marked as answer]
Put `vim.g.autoformat = false` in your `options.lua` file.

**Reply by radstevee** [reply]
Thanks!
